README.md

```diff
@@ -1,4 +1,23 @@
+This program will call the hcship.DestroyLayer system call on the specified folder
 
-run file:
+This will effectively reclaim a lot of hard-drive space for an undocumented reason.
 
-.\docker-ci-zap.exe -folder "C:\ProgramData\docker"
+# Usage (in admin PS console)
+
+1) Stop docker
+
+```powershell
+    net stop docker
+```
+2) run file:
+
+```powershell
+    cd "C:\ProgramData\docker"
+    .\docker-ci-zap.exe -folder "."
+```
```

Comment:
use the original instructions, for you should pass the full path in `-folder` argument (as it's unlikely to be different); however, putting the executable in your ProgramData folder isn't necessary, likely, nor recommend. people install tools in tools folders not data folders